explainer/explainer_pip_client.py

```diff
@@ -0,0 +1,31 @@
+import os
+import time
+
+from explainer.explainer_pipe_interface import FileSystemPipe
+
+POLL_INTERVAL = 10
+UPLOADS_DIR = os.path.join(__file__, "..", "..", "uploads")
+OUTPUTS_FOLDER = os.path.join(__file__, "..", "..", "outputs")
+
+
```

Comment:
the code is written in a good python syntax and there is a lot of investment in the code, but the logic is not so good, you are creating a new app engine at each call for send, 
also the web client doesn't take input from user, it's only for testing
you have two clients, cli client and web client and non of them takes input from the user,
you should read the exercise again

Reply:
sorry, I didnt understand this comment.
the web client is indeed for testing and doesn't take input from user. this is exactly what I did.
also the comment about the client is also unclear. The cli client is a way to communicate with the explanation in the easy way as it was needed in par1.
It can help with the debuging of the explanation but does not touch this part of the exercise